example/web/src/market/index.js

```diff
@@ -614,9 +615,9 @@ Market.prototype = {
     if (o.side === 'ASK') {
       bgColor = 'rgba(229, 85, 65, 0.3)';
     }
-    o.amount = parseFloat(bo.attr('data-amount')) - amount;
-    o.funds = parseFloat(bo.attr('data-funds')) - parseFloat(o.funds);
-    if (o.amount <= 0 || o.funds <= 0) {
+    o.amount = new BigNumber(bo.attr('data-amount')).minus(amount);
+    o.funds = new BigNumber(bo.attr('data-funds')).minus(o.funds);
```

Comment:
is o.funds already a BigNumber?

Reply:
http://mikemcl.github.io/bignumber.js/#minus
Minus object don't be a BigNumber